_source/_posts/2016-12-16-PMHC-MDS-Resources-Documentation-Update.md

```diff
@@ -10,7 +10,7 @@ The following documents have been updated:
 
 The following documents are now available:
 
-* [Guidance on client consent](https://www.pmhc-mds.com/doc/pmhc-consent-guidance-D16-1362220-20161202.pdf)
+* [Guidance on client consent](https://www.pmhc-mds.com/doc/pmhc-consent-guidance-20240430.pdf)

```

Comment:
Editing past posts needs to be done with carefully.  I'd prefer the edit makes it explicit the guidance has been updated since; linking to the consent section in the resources  (single source of truth).

Keep it simple:  "Guidance on client consent has been updated since 2016. See <link> for most recent guidance documentation."